submissions/sofiichuk/pop-up-task/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>pop-up task</title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<nav class="navBar">
+    <ul>
+        <li class="navBar__item_txt"><a href="#">Mail</a></li>
```

Comment:
Avoid moving sibling elements on hover or focus.

Reply:
Tried to set transparent border before changing on hover to non-transparent, tried ‘outline’ prop instead – both don’t work. I could move each element for a few pixels when hovered, but it’s kinda silly. Don’t know how to resolve this.